app/Http/Controllers/DesignerController.php

```diff
@@ -3,26 +3,29 @@
 namespace App\Http\Controllers;
 
 use Illuminate\Http\Request;
+use App\EvidencioAPI;
 
 class DesignerController extends Controller
 {
+
     /**
-     * Create a new controller instance.
+     * Show the application dashboard.
      *
-     * @return void
+     * @return \Illuminate\Http\Response
      */
-    public function __construct()
+    public function index()
     {
-        $this->middleware('auth');
+        return view('designer');
     }
 
     /**
-     * Show the application dashboard.
+     * Fetch
      *
-     * @return \Illuminate\Http\Response
      */
-    public function index()
+    public function fetchVariables()
```

Comment:
This is not really the Laravel-way of retrieving request variables. See for example https://laravel.com/docs/5.6/controllers and related pages.